### DISCUSSION TITLE
PPOCRLabel --lang ch  报错

### DISCUSSION THREAD

**Discussion by lemonlikelihood** [PaddlePaddle/PaddleOCR#12706] [Q&A]
运行PPOCRLabel --lang ch  报错
QObject::moveToThread: Current thread (0x5618cea016a0) is not the object's thread (0x5618d408e4e0).
Cannot move to target thread (0x5618cea016a0)

qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/home/xxx/.pyenv/versions/3.9.0/lib/python3.9/site-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb, eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl.
--------------------------------------
C++ Traceback (most recent call last):
--------------------------------------
No stack trace in paddle, may be caused by external reasons.

----------------------
Error Message Summary:
----------------------
FatalError: `Process abort signal` is detected by the operating system.
  [TimeInfo: *** Aborted at 1708934832 (unix time) try "date -d @1708934832" if you are using GNU date ***]
  [SignalInfo: *** SIGABRT (@0x32188dbb0000cff0) received by PID 53232 (TID 0x7f5f6e6b0740) from PID 53232 ***]

[1]    53232 abort (core dumped)  PPOCRLabel --lang ch

运行环境 ubuntu20，opencv_python-4.9.0.80，Qt版本5.12.8，python3.9

**Comment by Ataraxy33** [comment]
可以尝试重新安装opencv，或是尝试降低opencv版本至4.3.0.36

**Comment by TheHonestBob** [comment]
> 可以尝试重新安装opencv，或是尝试降低opencv版本至4.3.0.36

我真的吐了，paddle的依赖啥时候能够良好的解决呀，装paddle我得python3.11才能成功，装PPOCRLabel我得python3.9才能安装opencv版本至4.3.0.36，这不是矛盾嘛，环境安装太头疼了

**Comment by SWHL** [comment]
因PPOCRLabel已经搬家了，后续有问题请移步[PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel)提问哈！
此issue先行关闭了。
